### PR TITLE
Update eslint-config-seek to version 12.0

### PR DESCRIPTION
**Pull request by felixhabib**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
```
Comment (askoufis) [inline pnpm-lock.yaml:6]: You _might_ be on an older version of pnpm. Don't think this should've been removed.

**Comment by askoufis** [comment]
You might have a hard time upgrading the eslint config only in the root, and not the actual `sku` package. CI seems to agree.

File: pnpm-lock.yaml (hunk 1 above)
Comment (askoufis) [reply pnpm-lock.yaml:6]: Oh, did it just get moved to the bottom of the file? That's weird.

Comment (felixhabib) [reply pnpm-lock.yaml]: haha not sure, we think it's in the original place now tho

File: .changeset/gentle-buses-walk.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"sku": patch
+---
+
+Update eslint-config-seek to version 12.
```
Comment (askoufis) [inline .changeset/gentle-buses-walk.md:5]: IMO it would be good to have a message here mentioning that the majority of changes should be autofixable.

**Review by benjervis — APPROVED** [review]

**Pull request merged**